cli/options.go

```diff
@@ -335,7 +335,7 @@ func ProjectFromOptions(options *ProjectOptions) (*types.Project, error) {
 		} else {
 			opts.Name = filepath.Base(absWorkingDir)
 		}
-		opts.Name = regexp.MustCompile(`(?m)[a-z0-9]+[-_a-z0-9]*`).FindString(strings.ToLower(opts.Name))
+		opts.Name = normalizeName(opts.Name)
```

Comment:
I think the `strings.ToLower` has been removed, is it intentional?

Reply:
indeed, good catch

Reply:
Don’t we need it in the `nameFromEnv` case as well? Wouldn’t it be safer to add it to `normalizeName`?

Reply:
imho `nameFromEnv` should only apply to the "name from path" scenario (but this would require some significant refactoring to existing tests)
Setting project name by .env or command line flag, I would not expect the tooling to do any magic with it, and better fail with error if I did anything wrong